components/home/__tests__/homepage-telemedicine-cta.test.tsx

```diff
@@ -1,34 +1,17 @@
-import { render } from "@testing-library/react";
+import { render, screen } from "@testing-library/react";
 import { HomePageTelemedicineCTA } from "..//homepage-telemedicine-cta";
 
 describe("HomePageTelemedicineCta", () => {
   it("renders correctly", () => {
-    const { container } = render(<HomePageTelemedicineCTA />);
+    render(<HomePageTelemedicineCTA />);
+    const imgTelemedicine = screen.getByAltText(
+      "Telemedicine Gratis (Inisiatif Beberapa Dokter) - Cek Sekarang",
+    );
 
-    expect(container.firstChild).toMatchInlineSnapshot(`
-<section
-  class="bg-white overflow-hidden px-4 py-6 space-y-4 text-center"
->
-  <a
-    href="/telemedicine"
-  >
-    <div
-      style="display: block; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
-    >
-      <div
-        style="display: block; box-sizing: border-box; padding-top: 35.97560975609756%;"
-      />
-      <noscript />
-      <img
-        alt="Telemedicine Gratis (Inisiatif Beberapa Dokter) - Cek Sekarang"
-        data-nimg="true"
-        decoding="async"
-        src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
-        style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
-      />
-    </div>
-  </a>
-</section>
-`);
+    expect(imgTelemedicine).toBeInTheDocument();
+    expect(imgTelemedicine).toHaveAttribute(
+      "src",
+      "data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7",
+    );
```

Comment:
Instead of asserting something that's out of our control and is not meaningful enough to us, you can assert the anchor link here instead. You can use [this trick](https://github.com/kawalcovid19/wargabantuwarga.com/blob/85f7b0e0992cafcff406ad37ee30ff5b40c3872f/components/layout/__tests__/navigation.test.tsx#L153) to get the closest `a` element so that you can assert the `href` attribute afterwards.